src/boot/efi/part-discovery.c

```diff
@@ -37,14 +55,11 @@ static EFI_DEVICE_PATH *path_replace_hd(
         return ret;
 }
 
-static bool verify_gpt(union GptHeaderBuffer *gpt_header_buffer, EFI_LBA lba_expected) {
-        EFI_PARTITION_TABLE_HEADER *h;
+static bool verify_gpt(GptHeader *h, EFI_LBA lba_expected) {
```

Comment:
maybe add a comment that the first argument is kinda const, but also not

Reply:
Not addressed yet.